test/unit/signal_wait_until.c

```diff
@@ -65,15 +65,16 @@ int main(int argc, char *argv[])
     }
 
     uint64_t retval = shmem_signal_wait_until(&sig_addr, SHMEM_CMP_LE, npes);
-    if (retval > npes)
+
+    if (retval > (uint64_t)npes)
```

Comment:
Nitpick.. Do we use a space between `(type)` and `operand` usually?

Reply:
Huh, I see we're inconsistent with the type-cast space throughout SOS (e.g., [here](https://github.com/Sandia-OpenSHMEM/SOS/blob/v1.5.0/src/data_c.c4#L439) vs. [here](https://github.com/Sandia-OpenSHMEM/SOS/blob/v1.5.0/src/transport_ofi.h#L1201)), but my hunch is we usually include a space.

I'll add the space here and try to be consistent about this style in the future.